pyspecProcScripts/phasing.py

```diff
@@ -753,7 +754,7 @@ def hermitian_function_test(
         return echo_peak + min_echo
 
 
-def determine_sign(s, direct="t2", fl=None):
+def determine_sign(s, direct="t2", basename=None, fl=None):
```

Comment:
Rather, below replace the "if basename" with `if not hasattr(fl,"basename") or fl.basename is None`